examples/bert/main.py

```diff
@@ -18,6 +18,46 @@
                                       build_optimizer, build_scheduler)
 
 
+def calculate_batch_size_info(global_batch_size, device_microbatch_size):
```

Comment:
I think it might be worthwhile to move both these functions to `common/...` and then import them, in both the `bert` and `llm` examples!